### ISSUE TITLE
content not loading in amp-accordian if followed amp-list with amp-iframe

### ISSUE THREAD

**Issue by Rajashree-sys** [ampproject/amphtml#24708]
## What's the issue?

An amp page with many amp-list's followed by amp-accordian holds many amp-list's. amp-iframe is there inbetween these two set. Because of that, amp-accordian is not getting loaded with content by default while opening it in browser, once user clicks on that it is getting loaded with data.
When we remove the amp-iframe section inbetween, it is solving the problem. Can we know what the problem is, and how can we overcome this.
## How do we reproduce the issue?

Please provide a public URL and ideally a reduced test case (e.g. on jsbin.com) that exhibits only your issue and nothing else. Provide step-by-step instructions for reproducing the issue:

1. Have 4 amp-list continuously
2. Have amp-accordian with amp-list's inside it
3. In between point 1 and 2, Have section with amp-iframe wrapped with amp-lightbox
4. Check that amp-accordian is loaded with json content, when hitting the page for first time, without clicking on the accordian

## What browsers are affected?

All browser in mobile device

## Which AMP version is affected?

**amp.js:157** 
Powered by AMP ⚡ HTML – Version 1909141409590

**Comment by nainar** [comment]
Dupe of https://github.com/ampproject/amphtml/issues/23449

**Comment by Rajashree-sys** [comment]
> Dupe of #23449

I checked #23449. I feel it is not the same issue. We have difference like, in this scenario accordian is not getting loaded because of the iframe sitting above that accordian section. But when we click on the accordian content is getting loaded.
I have attached files to replicate my scenario.
Also attaching the screen shot to differentiate(accordian not loading, and accordian loading on click)

<img width="948" alt="Accordian-ContentNotLoading" src="https://user-images.githubusercontent.com/55743352/66202017-a9d36180-e69c-11e9-82d7-c3a9513479bb.png">
<img width="945" alt="Accordian-ContentLoadingOnClick" src="https://user-images.githubusercontent.com/55743352/66202031-ac35bb80-e69c-11e9-89f9-eca83c0c36d9.png">


[google-amp-pages.zip](https://github.com/ampproject/amphtml/files/3690141/google-amp-pages.zip)
